tests/saliency_metrics/metrics/insertion_deletion/test_insertion_deletion_perturbation.py

```diff
@@ -0,0 +1,60 @@
+from typing import List
+
+import numpy as np
+import pytest
+import torch
+
+from saliency_metrics.metrics.insertion_deletion import ProgressivePerturbation
+
+
+@pytest.fixture
+def dummy_img_and_smap():
```

Comment:
Please create a file `tests/saliency_metrics/metrics/insertion_deletion/conftest.py` and move this fixture to the file, so that this fixture can be shared by all the test files under the same folder.